### PR TITLE
More fix for 4.11 ironic images jobs

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

File: core-services/image-mirroring/openshift/mapping_origin_4_11 [hunk 1]
```diff
@@ -143,6 +143,8 @@ registry.ci.openshift.org/origin/4.11:ibm-vpc-node-label-updater quay.io/openshi
 registry.ci.openshift.org/origin/4.11:ibmcloud-machine-controllers quay.io/openshift/origin-ibmcloud-machine-controllers:4.11 quay.io/openshift/origin-ibmcloud-machine-controllers:4.11.0 quay.io/openshift/origin-ibmcloud-machine-controllers:latest
 registry.ci.openshift.org/origin/4.11:image-customization-controller quay.io/openshift/origin-image-customization-controller:4.11 quay.io/openshift/origin-image-customization-controller:4.11.0 quay.io/openshift/origin-image-customization-controller:latest
 registry.ci.openshift.org/origin/4.11:insights-operator quay.io/openshift/origin-insights-operator:4.11 quay.io/openshift/origin-insights-operator:4.11.0 quay.io/openshift/origin-insights-operator:latest
+registry.ci.openshift.org/origin/4.11:ironic quay.io/openshift/origin-ironic:4.11 quay.io/openshift/origin-ironic:4.11.0 quay.io/openshift/origin-ironic:latest
+registry.ci.openshift.org/origin/4.11:ironic-agent quay.io/openshift/origin-ironic-agent:4.11 quay.io/openshift/origin-ironic-agent:4.11.0 quay.io/openshift/origin-ironic-agent:latest
```
Comment (bparees) [inline core-services/image-mirroring/openshift/mapping_origin_4_11:147]: @smarterclayton do you have opinions about prefixing this with `baremetal`?  that would be my inclination, unless it is also used for other platforms.

Comment (bparees) [reply core-services/image-mirroring/openshift/mapping_origin_4_11:147]: well on further analysis i see we've already been using these names (no baremetal prefix) in 4.10, so i guess we should leave it alone.

**Review by iurygregory — APPROVED** [review]
/lgtm

**Pull request merged**